server/lib/oauth/index.ts

```diff
@@ -19,20 +20,16 @@ class CustomTokenHandler extends TokenHandler {
   }
 
   getTokenType = function (model) {
-    return {
-      valueOf: () => {
-        const accessToken = model.user.jwt(
-          {
-            scope: 'oauth',
-            // eslint-disable-next-line camelcase
-            access_token: model.accessToken,
-          },
-          auth.TOKEN_EXPIRATION_SESSION_OAUTH, // 90 days,
-        );
+    const accessToken = model.user.jwt(
+      {
+        scope: 'oauth',
         // eslint-disable-next-line camelcase
-        return { access_token: accessToken };
+        access_token: model.accessToken,
       },
-    };
+      auth.TOKEN_EXPIRATION_SESSION_OAUTH, // 90 days,
+    );
+
+    return BearerTokenType(accessToken, auth.TOKEN_EXPIRATION_SESSION_OAUTH, null, model.scope);
```

Comment:
If you wanted to (and you actually support refresh tokens) then you could do the same JWT thing as above pretty much, except with `model.refreshToken` and the `refreshTokenLifetime` value which is just [hardcoded](https://github.com/opencollective/opencollective-api/blob/57a84179df3f59f078715a58ccc315171527a18b/server/lib/oauth/index.ts#L80) to `60 * 60 * 24 * 365`, and then replace the `null` with that refresh token JWT.